sklego/pipeline.py

```diff
@@ -0,0 +1,289 @@
+'''
+Pipelines, variances to the :class:`sklearn.pipeline.Pipeline` object.
+'''
+
+
+import logging
+import datetime as dt
+
+from sklearn.pipeline import Pipeline
+from sklearn.utils.validation import check_memory
+
+
+def default_log_callback(output, execution_time, **kwargs):
```

Comment:
should this function be private?

Reply:
Made it public such that it is visible in the docs.